src/lab/Graphs/LegendTable/LegendTable.tsx

```diff
@@ -83,4 +72,17 @@ const LegendTable: React.FC<LegendProps> = ({
     </TableContainer>
   );
 };
+
+const LegendTable: React.FC<LegendTableProps> = ({ ...rest }) => {
+  return (
+    <ParentSize>
```

Comment:
Can this be done with width: 100% in the LegendTable like we have discussed previously?

Reply:
sure